index.js

```diff
@@ -242,6 +242,35 @@ function build (schema, options) {
   ajvInstance = new Ajv({ ...options.ajv, strictSchema: false, uriResolver: fastUri })
   ajvFormats(ajvInstance)
 
+  const validateDateTimeFormat = ajvFormats.get('date-time').validate
+  const validateDateFormat = ajvFormats.get('date').validate
+  const validateTimeFormat = ajvFormats.get('time').validate
+
+  ajvInstance.addKeyword({
+    keyword: 'fjs_date_type',
+    validate: (schema, date) => {
+      if (date && typeof date.toISOString === 'function') {
```

Comment:
As I understand, fjs not specifies that date should be the Date type object. It says that it should be an object with toISOString or format method.

Reply:
No, it says it should be a string or a `Date` object, where in the case of the `Date` object it will be serialized through the `.toISOString` method -- https://github.com/fastify/fast-json-stringify#specific-use-cases

Simply because an object has a `.toISOString` method does not mean it is a valid `Date` object. I showed this above. So if there is going to be code to verify that an object is a valid `Date` object instance, it should be more complete.

Reply:
https://github.com/fastify/fast-json-stringify/blob/074a120dcdaa97759f2df9890a3f9708a7e1cba0/index.js#L130